torch/csrc/stub.c

```diff
@@ -9,7 +9,11 @@ extern "C"
 __attribute__((visibility("default"))) PyObject* PyInit__C(void);
 #endif
 
+extern "C" TORCH_API void global_kineto_init();
```

Comment:
I don't think you need the extern "C" here that is for cpp only

Reply:
Yeah, that's true,  'extern' is enough for function declaration in c files.